samples/README.md

```diff
@@ -4,65 +4,171 @@ This repository contains a few samples to help you get started quickly with the
 
-In order to test the sample applications you need to have access to a Kafka instance. We recommend using the Confluent Docker Compose sample to get started with a local Kafka and data generator.
+In order to test the sample applications you need to have access to a Kafka instance. We have a several way to enable it.
+
+### Visual Studio Core Remote - Containers
+Several samples provides [DevContainer](https://github.com/microsoft/vscode-dev-containers) configuration. If you start the [Visual Studio Code](https://code.visualstudio.com/) on the target sample directory, it will automatically start a development environment on a Docker container with a local kafka cluster. It is the easiest option for starting a kafka cluster. [Developing inside a Container](https://code.visualstudio.com/docs/remote/containers)
+
+### Confluent Docker Compose
+
+We provide the Confluent Docker Compose sample to get started with a local Kafka and data generator.
 Follow the guide at https://docs.confluent.io/current/quickstart/ce-docker-quickstart.html#cp-quick-start-docker.
 
 Make sure you complete the steps at least until the topics pageviews, users and pageviews_female are created (including data generators). The included .NET sample function contains a consumer for each of those 3 topics.
 
-## Using the .NET Sample Function
+### Confluent Cloud
+
+[Confluent Cloud](https://www.confluent.io/confluent-cloud/?_ga=2.64661359.1523477604.1592516088-840657119.1591307401) is a fully managed, cloud-native event stream platform powered by Apatch Kafka. The samples includes Confluence Cloud samples to understand the real-world configration.
+
+## Language Support
+
+Azure Functions Kafka Extension support several languages with follwing samples. For more details and getting started, please refer the links below.
+
+| Language | Description | Link | DevContainer |
+| -------- | ----------- | ---- | ------------ |
+| C# | C# precompiled sample with Visual Studio | [Readme](dotnet/README.md)| No |
+| Java | Java 8 sample | [Readme](java/README.md) | Yes |
+| JavaScript | Node 12 sample | [Readme](javascript/README.md)| Yes |
+| PowerShell | PowerShell 6 Sample | [Readme](powershell/README.md)| No |
+| Python | Python 3.8 sample | [Readme](python/README.md)| Yes |
+| TypeScript | TypeScript sample (Node 12) | [Readme](typescript/README.md)| Yes |
+
+## Custom Container
+
+Custom container enable us to deploy custom container to the Function App. We can use any other languages, however, as an example, we provide a java sample to explan how to develop it.
+
+| Language | Description | Link | DevContainer |
+| -------- | ----------- | ---- | ------------ |
+| Java | Custom container sample | [Readme](container/README.md)| No |
+
+## Tips
+
+Kafka extension supports several lanugages, however, it uses the same Azure Functions host. For this reason, there is a common configuration for each lanugages. I'd like to provide configuration tips in common.
```

Comment:
Change "I'd ...." to "Please find below some common notes which apply to all the languages."

Reply:
Done!